analyze/aps-dnc.c

```diff
@@ -2629,7 +2609,7 @@ void print_instance(INSTANCE *i, FILE *stream) {
 
 void print_edge_helper(DEPENDENCY kind, CONDITION *cond, FILE* stream) {
   if (stream == 0) stream = stdout;
-  if ((kind & DEPENDENCY_MAYBE_SIMPLE) == DEPENDENCY_MAYBE_SIMPLE) fputc('s', stream);
+  if (!(kind & DEPENDENCY_MAYBE_SIMPLE)) fputc('o', stream);
```

Comment:
Yes, print an 'o' and then SET the MAYBE_SIMPLE bit so the next switch will do something else.  ANd don't print an X.